### ISSUE TITLE
Late-breaking glTF projects

### ISSUE THREAD

**Issue by pjcozzi** [KhronosGroup/glTF#1058]
This issue is for tracking late-breaking glTF ecosystem projects that haven't yet made it to the [master list](https://github.com/KhronosGroup/glTF#contents) in this repo's README.md.

To contribute, feel free to chime in here with projects you are aware of / working on, and open pull requests to update the master list.

_(The checkboxes are used to keep track of when something is added to the master list)._

_(Everything is glTF 2.0 unless otherwise noted)._

**Comment by pjcozzi** [comment]
* [ ] Sketchup warehouse import/export extension. https://github.com/KhronosGroup/glTF/issues/1051#issuecomment-320078379

via @CentaurMare

**Comment by pjcozzi** [comment]
* [ ] Unity HoloLens Tooklit. https://github.com/Microsoft/HoloToolkit-Unity/pull/793

via @keveleigh

**Comment by pjcozzi** [comment]
* [ ] Qt 3D. https://twitter.com/sean_harmer/status/890247512971501569

**Comment by pjcozzi** [comment]
* [ ] Metal renderer. https://twitter.com/warrenm/status/891558755657175040

via @warrenm

**Comment by pjcozzi** [comment]
* [ ] PiXYZ export. https://twitter.com/PixyzSoft/status/892022576851169280

**Comment by pjcozzi** [comment]
* [ ] Blender importer. https://github.com/ksons/gltf-blender-importer

via @ksons

**Comment by pjcozzi** [comment]
* [ ] Godot. https://twitter.com/reduzio/status/892210751338930177, https://github.com/godotengine/godot/issues/7952

via @reduz

**Comment by pjcozzi** [comment]
* [ ] React VR plans to support glTF. https://twitter.com/pjcozzi/status/837508186487676928 (via GDC meetup)

**Comment by pjcozzi** [comment]
* [ ] assetkit. https://github.com/recp/assetkit

via @recp

**Comment by pjcozzi** [comment]
* [ ] Flatbuffers. https://github.com/KageKirin/flatgltf_2_0

via @KageKirin

**Comment by pjcozzi** [comment]
* [ ] gltf-bounding-box: Node module to compute asset's bounding box. https://www.npmjs.com/package/gltf-bounding-box

via @risq

**Comment by pjcozzi** [comment]
* [ ] Chromium. https://groups.google.com/a/chromium.org/forum/#!topic/graphics-dev/NxG1JunmBO8 and https://codereview.chromium.org/2852533004

glTF 1.0 and Binary glTF.  Would be great for them to consider glTF 2.0.

CC @toji

**Comment by pjcozzi** [comment]
* [ ] gltfviewer: https://github.com/virtualgis/gltfviewer, https://www.virtualgis.io/gltfviewer/

Built on Cesium, needs to update to Cesium 1.36+ to support glTF 2.0

via @pierotofy

**Comment by pjcozzi** [comment]
* [ ] Interest from servo. https://github.com/servo/servo/issues/15506

via @cvan

**Comment by pjcozzi** [comment]
* [ ] Interest from regl. https://github.com/regl-project/regl/issues/186

via @mikolalysenko

**Comment by pjcozzi** [comment]
* [ ] FME import/export. https://blog.safe.com/2017/01/whats-new-in-fme-2017-0-release-candidate-available/

glTF 1.0?

**Comment by pjcozzi** [comment]
* [ ] glTF loader in NXT prototype (not up-to-date). https://github.com/gpuweb/nxt-standalone

via @kainino0x

**Comment by pjcozzi** [comment]
* [ ] Semantic diff/merge in-progress, https://github.com/KhronosGroup/glTF/issues/867#issuecomment-287424242

via @xelatihy

**Comment by pjcozzi** [comment]
* [ ] Subreddit: https://www.reddit.com/r/glTF/

If someone is willing to help post content here, we could promote this more widely.  Please chime in here if you can help.

**Comment by reduz** [comment]
Would be fantastic, in case no one volunteers, if Khronos could at least secure financing to provide export plugins for Autodesk software..

**Comment by pjcozzi** [comment]
* [ ] Autodesk Forge import

via @nopjia

**Comment by pjcozzi** [comment]
@reduz for sure, I agree!!!

**Comment by pierotofy** [comment]
@pjcozzi https://www.virtualgis.io/gltfviewer/ is now updated to use Cesium 1.36, which supports glTF 2.0. Thanks!

**Comment by pjcozzi** [comment]
@pierotofy great, feel free to open a pull request to add it to the [master list](https://github.com/KhronosGroup/glTF#contents).

**Comment by pjcozzi** [comment]
* [ ] @Marlamin's implementation, https://twitter.com/marlamin/status/894235053299036160

**Comment by pjcozzi** [comment]
* [ ] glTF interest from JME, https://hub.jmonkeyengine.org/t/gltf-support-for-jme/39117

via @tonihele

**Comment by CentaurMare** [comment]
I'm still waiting on the Sketchup Extension Warehouse team to approve my glTF exporter and glTF importer for Sketchup. The issue has been its their summer vacation. (Crazy people taking their summer vacation in winter! ;)  Meanwhile I've added support for 8-bit indices, something I missed earlier - which really helps bring the file size down in some cases.

**Comment by pjcozzi** [comment]
* [ ] VRML to glTF, e.g., https://github.com/KhronosGroup/glTF/issues/1046

via @nbouayad

**Comment by pjcozzi** [comment]
* [ ] three.js glTF 2.0 exporter, https://twitter.com/mrdoob/status/897469523900547073

via @fernandojsg @mrdoob

**Comment by pjcozzi** [comment]
* [ ] C++ obj2gltf, https://github.com/xelatihy/yocto-gl/blob/master/apps/yobj2gltf.cpp

via @xelatihy

**Comment by pjcozzi** [comment]
* [ ] nunuStudio export (built on three.js), https://twitter.com/tentonej/status/897564345344851968

**Comment by pjcozzi** [comment]
* [ ] NuGet for Net.Standard, https://github.com/KhronosGroup/glTF-CSharp-Loader/issues/2#issuecomment-322745817

@vpenades

**Comment by CentaurMare** [comment]
My Sketchup extensions for glTF import and export are now published in the Sketchup Extension Warehouse.

http://extensions.sketchup.com/en/content/gltf-exporter
http://extensions.sketchup.com/en/content/gltf-import


Exporter Limitations: Exports polygon meshes as triangles (primitive-mode 4), no lines, triangle strips or triangle fans. Choice of embedded glTF, binary glTF, and Microsoft-Compatible binary glTF for exporting to Paint3D.

Importer Limitations: Will not handle 8 or 16 bit normalized UVs (only floats). Only primitive mode 4 triangles will be imported.
The importer is very slow, (so please be patient) as it is written in Ruby and the ruby API for Sketchup is fairly limited when it comes to importing geometry. I may need to start on a C++ version of the importer, but this should do for now.

I don't know how to 'pull request to update the master list' so please help me out.

**Comment by pjcozzi** [comment]
Awesome news, congrats @CentaurMare!  Added to the master list in #1076.

What's next for you with glTF? 😄

**Comment by CentaurMare** [comment]
What's next for me with glTF?

I will be monitoring the import and export extensions for a while, fixing bugs (already one has been discovered, I'm not setting transparency properly during import), and maybe adding support for importing and exporting lines.

https://forums.sketchup.com/t/gltf-import-and-export-webgl-paint3d/50128


My next big glTF project will probably be a re-write of the Importer from ruby code into C++ code to help speed things along, as I don't think waiting hours for a model to import into Sketchup is going to do much to encourage Sketchup users to adopt the file format! The speed issue isn't due to Ruby or my poor code, but is related to the API presented to ruby from Sketchup. Well, that is my excuse anyway.

**Comment by pjcozzi** [comment]
@CentaurMare cool stuff, thanks for sharing!  C++ port sounds super important for adoptions.

**Comment by pjcozzi** [comment]
* [ ] [freeciv-web](https://github.com/freeciv/freeciv-web) via @andreasrosdal (https://github.com/KhronosGroup/glTF-Blender-Exporter/issues/76)

**Comment by pjcozzi** [comment]
* [ ] https://plexearth.com/ using glTF 1.0 and plans to upgrade to 2.0

**Comment by fernandojsg** [comment]
fwiw just published a post about our support for the gltf exporter in three.js and a-frame https://blog.mozvr.com/gltf-exporter-in-three-js-and-a-frame/

**Comment by pjcozzi** [comment]
@fernandojsg really fantastic article, thanks for sharing!  I added it to the list of glTF articles in #1084.  Please keep us posted on your work!

**Comment by pjcozzi** [comment]
* [ ] [glTF-Asset-Generator](https://github.com/bghgary/glTF-Asset-Generator) via @bghgary and @sbtron

**Comment by pjcozzi** [comment]
* [ ] Binary glTF loader by @jonathansty

https://twitter.com/jonathansty/status/904850392705589248

**Comment by pjcozzi** [comment]
* [ ] glTF support in game engine by @udoprog

https://github.com/KhronosGroup/glTF-Blender-Exporter/issues/39#issuecomment-326976200

**Comment by pjcozzi** [comment]
* [ ] glTF loader in [code4game](https://github.com/code4game/libgltf) via @alexchicn

https://twitter.com/c4gio/status/907267375392624641

**Comment by pjcozzi** [comment]
* [ ] WebGL engine by @Inkthorne

https://twitter.com/inkthorne/status/906944549704015873

**Comment by pjcozzi** [comment]
* [ ] BabylonJS' Max exporter can export static glTF assets
   * Documentation: http://doc.babylonjs.com/exporters/3dsmax
   * Sources: https://github.com/BabylonJS/Babylon.js/tree/master/Exporters/3ds%20Max
   * Link to binaries: https://github.com/BabylonJS/Babylon.js/blob/master/Exporters/3ds%20Max/Max2Babylon-0.15.0.zip

**Comment by pjcozzi** [comment]
* [ ] Substance Painter 2017.3 glTF 2.0 export! https://www.allegorithmic.com/blog/new-substance-painter-autosave-gltf-and-felix-exports

**Comment by pjcozzi** [comment]
* [ ] https://github.com/ux3d/2gltf2 - Blender/COLLADA/FBX/OBJ -> glTF 2.0

@UX3D-nopper when ready, feel free to open a pull request to add this to the tools list in this repo's README.md

**Comment by pjcozzi** [comment]
* [ ] https://github.com/gltf-rs/gltf-validator-web - Experimental glTF validator written in Rust and compiled to WebAssembly by @bwasty

@lexaknyazev @bwasty any potential collaboration opportunities here around https://github.com/KhronosGroup/glTF-Validator?

**Comment by pjcozzi** [comment]
* [ ] glTF for UE4. mesh import. https://c4gio.itch.io/gltf-for-ue4

**Comment by pjcozzi** [comment]
* [ ] [GLTFSceneKit](https://github.com/magicien/GLTFSceneKit) via @magicien

**Comment by pjcozzi** [comment]
* [ ] React VR 2 supports glTF. https://twitter.com/immediatedelay/status/915687927924924416

(Older comment: React VR plans to support glTF. https://twitter.com/pjcozzi/status/837508186487676928 - via GDC meetup)

**Comment by pjcozzi** [comment]
* [ ] [UnityGLTF](https://github.com/KhronosGroup/UnityGLTF) - in-progress glTF import/export for Unity

**Comment by pjcozzi** [comment]
* [ ] [qtek-model-viewer](https://github.com/pissang/qtek-model-viewer) - Simple 3D model viewer based on QTEK - via @pissang
   * https://twitter.com/pissang1/status/918453589588131840

**Comment by pjcozzi** [comment]
* [ ] Cross-platform desktop glTF preview, https://github.com/donmccurdy/three-gltf-viewer, by @donmccurdy 
   * https://twitter.com/donrmccurdy/status/919363023382331392

@donmccurdy can you add a link to this repo's README.md?

**Comment by pjcozzi** [comment]
* [ ] [kepler3d](https://github.com/dgough/kepler3d) - OpenGL and C++11 game engine that loads glTF 2.0 - by @dgough

**Comment by pjcozzi** [comment]
* [ ] [lazy-gltf2](https://github.com/dgough/lazy-gltf2) - Header only C++11 glTF 2.0 loader - by @dgough

**Comment by pjcozzi** [comment]
* [ ] [gltf2-webgl](https://github.com/topfs2/gltf2-webgl) - gltf2 loading and rendering experimentation in webgl - by @topfs2

**Comment by pjcozzi** [comment]
* [ ] [code-vr](https://github.com/OpenHID/code-vr) - Program and explore real applications with virtual reality! - by @alaingalvan and @lborg019

**Comment by pjcozzi** [comment]
* [ ] [gltf-objc](https://github.com/warrenm/gltf-objc) - An Objective-C glTF 2.0 loader and Metal-based renderer - by @warrenm

**Comment by pjcozzi** [comment]
* [ ] [rome-gltf](https://github.com/mrdoob/rome-gltf)

@donmccurdy should we add a link to this from glTF-Sample-Models?

**Comment by pjcozzi** [comment]
* [ ] [coronal](https://github.com/hyperfuse/coronal) - A modern 3D rendering library for quickly building games and apps - by @alaingalvan

glTF 2.0 in progress.

**Comment by pjcozzi** [comment]
* [ ] [TriLib](https://www.assetstore.unity3d.com/en/#!/content/91777) - Unity import/export using assimp via @kimkulling
   * https://twitter.com/KimKulling/status/920637484173811716

**Comment by pjcozzi** [comment]
* [ ] [qtek-model-viewer](https://github.com/pissang/qtek-model-viewer) - glTF2.0 model viewer based on QTEK - by @pissang
   * https://twitter.com/pissang1/status/921295918934122497

Wow!

**Comment by pjcozzi** [comment]
* [ ] Verge3D plans to supported glTF 2.0. https://twitter.com/soft8soft/status/920600876867301376

**Comment by pjcozzi** [comment]
* [ ] FBX-to-glTF converter by @chipweinberger, #1143

May become open source: https://github.com/KhronosGroup/glTF/issues/1143#issuecomment-341889343

**Comment by pjcozzi** [comment]
* [ ] Playcanvas glTF loader, https://twitter.com/willeastcott/status/930271384814592000

**Comment by pjcozzi** [comment]
* [ ] [gltf_scenekit](https://github.com/3D4Medical/gltf_scenekit) by @sakrist

**Comment by donmccurdy** [comment]
glTF loader for [regl](http://regl.party/):

https://github.com/gamedev-js/regltf

^tested this out and [had some trouble](https://github.com/gamedev-js/regltf/issues/1), but the project looks promising.

**Comment by pjcozzi** [comment]
* [ ] [Hilo3d](https://github.com/hiloteam/Hilo3d) by @06wj

**Comment by pjcozzi** [comment]
* [ ] [macOS QuickLook plugin](https://github.com/magicien/GLTFQuickLook) by @magicien

https://github.com/KhronosGroup/glTF-Sample-Models/issues/101#issuecomment-346158492

**Comment by pjcozzi** [comment]
* [ ] [AMD Compressonator](https://gpuopen.com/compressonator-v2-7-release-adds-cross-platform-support-3d-model-compression-gltf-v2-0/)

**Comment by pjcozzi** [comment]
* [ ] Maya2glTF by @Ziriax
   * develop branch: https://github.com/Ziriax/Maya2glTF/tree/develop

**Comment by pjcozzi** [comment]
* [ ] Google Poly API glTF export: https://twitter.com/donrmccurdy/status/936301152953499648

via @donmccurdy

**Comment by pjcozzi** [comment]
* [ ] [glTF-Toolkit](https://github.com/Microsoft/glTF-Toolkit) - a collection of tools for modifying and optimizing glTF assets

**Comment by pjcozzi** [comment]
* [ ] [Simpolygon Cloud](https://www.neowin.net/news/microsoft-reveals-simplygon-cloud-to-aid-mixed-reality-development) - optimizing geometry and textures for MR

@sbtron do you want to add this to the main README.md?

**Comment by pjcozzi** [comment]
* [ ] [Draco inspector](https://github.com/InSimo/three-gltf-viewer/blob/tools/tools/DracoInspector.js) by @JeremieA

via https://github.com/KhronosGroup/glTF/issues/1114#issuecomment-350043164

**Comment by sbtron** [comment]
- [ ] [Maya exporter](https://github.com/BabylonJS/Exporters/tree/master/Maya)
       Still early work in progress.   Feedback welcome.

**Comment by pjcozzi** [comment]
* [ ] https://gltf.insimo.com/ - UI combining many tools: validation, gltf-pipeline, etc. by @insimo

via https://twitter.com/JeremieAllard/status/939279286908936192

**Comment by pjcozzi** [comment]
* [ ] [gltf-boundingsphere](https://github.com/iWun/gltf-boundingsphere) by @iWun

**Comment by pjcozzi** [comment]
* [ ] [gltf-bounding-box](https://github.com/iWun/gltf-bounding-box) by @iWun

Also see https://github.com/KhronosGroup/glTF/issues/1058#issuecomment-320460384

**Comment by pjcozzi** [comment]
* [ ] [gltf2](https://github.com/urish/gltf2) by @urish

**Comment by pjcozzi** [comment]
* [ ] [glTF Shell Extensions](https://github.com/bghgary/glTF-Shell-Extensions)

@bghgary do you want to add this to the README.md?

**Comment by bghgary** [comment]
@pjcozzi Let's leave it out of the README for now. Still trying to figure out a better deployment story. Also seeing issues with virus scanner detecting false positives on the executable.

**Comment by pjcozzi** [comment]
* [ ] OTOY Autodesk SDK FBX -> glTF plugin

https://twitter.com/OTOY/status/951643625795956742

**Comment by pjcozzi** [comment]
* [ ] glTF Unity importer, https://github.com/ousttrue/UniGLTF by @ousttrue via https://github.com/KhronosGroup/glTF/issues/456#issuecomment-359259376

**Comment by pjcozzi** [comment]
* [ ] jgltf-viewer-android by @javagl

via https://github.com/KhronosGroup/glTF/issues/1218#issuecomment-359941145

**Comment by pjcozzi** [comment]
* [ ] [Unreal Engine 4.19 preview has experimental glTF import](https://forums.unrealengine.com/development-discussion/content-creation/1415582-the-new-gltf-import-supported-in-4-19-preview-is-awesome-experimentation)

**Comment by pjcozzi** [comment]
* [ ] [glTFForUE4](https://github.com/code4game/glTFForUE4) - glTF for Unreal Engine 4 by @alexchicn

(separate from https://github.com/KhronosGroup/glTF/issues/1058#issuecomment-361947482 AFAIK)

**Comment by alexchicn** [comment]
@pjcozzi It is good news. I am glad to see that the official supports the glTF format. 👍 

It likes a confliction between their plugin and my plugin. But my plugin will have a lot of features in the plan. For example:

| Features | The official `glTF Importer` | My [`glTF for UE4`](https://github.com/code4game/glTFForUE4) |
|:------------:|:------------:|:------------:|
| Support Versions | From UE4.19 | From UE4.10 💪 |
| Can Import | Yes | Yes |
| Can Export | No | Yes(doing) |
| Work in Editor | Yes | Yes |
| Work in Runtime | No | Yes(doing) |
| Generate Material | Yes | Yes(doing) |
| ... | ??? | Yes 😄 |

> I use an official code as a reference to optimise my plugin. 😃 

Thanks

**Comment by pjcozzi** [comment]
@alexchicn wow, thanks for the insight.  If you think there is potential collaboration with the UE glTF dev, I am happy to introduce you.

**Comment by alexchicn** [comment]
@pjcozzi Thanks. I will consider the collaboration.

**Comment by pjcozzi** [comment]
@alexchicn OK, I send you two a note now.

**Comment by pjcozzi** [comment]
* [ ] [Vulkan-glTF-PBR](https://github.com/SaschaWillems/Vulkan-glTF-PBR) by @SaschaWillems

**Comment by alexkowel** [comment]
Verge3D, a Three.js-based framework uses glTF 2.0 by default. Includes exporters for [Blender](https://www.soft8soft.com/verge3d-1-0-released/) and [3ds Max](https://www.soft8soft.com/verge3d-for-3ds-max-released/). Non-standard features are implemented on top of vendor extensions prefixed with **S8S** (_S8S_v3d_data_, _S8S_v3d_material_data_, _S8S_v3d_camera_data_ ... etc ).

**Comment by pjcozzi** [comment]
Thanks @alexchicn, I think we have Verge3D info listed above, https://github.com/KhronosGroup/glTF/issues/1058#issuecomment-339306182

**Comment by alexkowel** [comment]
Yep, my bad! BTW i just filed a [request](https://github.com/KhronosGroup/glTF/issues/1240) to add S8S prefix and also added [Verge3D](https://github.com/KhronosGroup/glTF/pull/1241) to the Readme file. Hopefully you consider them for merging.

**Comment by pjcozzi** [comment]
* Unbound glTF export

https://twitter.com/abductee_org/status/959644076747337729

**Comment by SaschaWillems** [comment]
>  Vulkan-glTF-PBR by @SaschaWillems
Just pushed a first working version.

**Comment by pjcozzi** [comment]
* [ ] [glb-packer](https://glb-packer.glitch.me/) by @msfeldstein

via https://twitter.com/msfeldstein/status/963907497906991104

**Comment by pjcozzi** [comment]
* [ ] Facebook 3D posts
* [ ] Sony 3D Creator
* [ ] Oculus Medium
* [ ] Modo
* [x] Google Poly (already mentioned above, https://github.com/KhronosGroup/glTF/issues/1058#issuecomment-349661876)


See
* https://uploadvr.com/facebook-3d-posts-newsfeed-gltf/
* https://developers.facebook.com/docs/sharing/3d-posts
* https://techcrunch.com/2018/02/20/facebook-3d-posts/

**Comment by pjcozzi** [comment]
* [ ] AssetForge glTF export, https://twitter.com/KenneyNL/status/967182458415435777

via @KenneyNL

**Comment by pjcozzi** [comment]
* [ ] AMD Radeon ProRender Blender plugin, https://pro.radeon.com/en/radeon-prorender-transforms-rendering-workflows-interactive-viewport-denoising/

via @erich666

**Comment by pjcozzi** [comment]
via @nickkraakman:

> Hi, we're working on integrating glTF conversion into a microservice framework, so we built upon the amazing ClayGL fbx2gltf converter (which also support OBJ and COLLADA inputs), and wrapped it in a web API.
>
>The API can be tested live here at https://gltfapi.co
>
>The source code is published on GitHub here: https://github.com/headjack-dev/gltf-api

https://github.com/KhronosGroup/glTF/issues/456#issuecomment-369896136

**Comment by pjcozzi** [comment]
* [ ] [gltf-loader-ts](https://github.com/bwasty/gltf-loader-ts) - Engine-agnostic glTF 2.0 Loader in TypeScript - by @bwasty

**Comment by pjcozzi** [comment]
* [ ] Convert from FBX, OBJ, GLTF, GLB, Collada ( DAE ) to GLTF or GLB in the browser by @looeee.

https://twitter.com/looeeeb/status/971060724611952640

**Comment by pjcozzi** [comment]
* [ ] vectary - export .glb for Facebook 3D posts

https://twitter.com/milangladis/status/969137248598929409

via @milangladis

**Comment by donmccurdy** [comment]
* [ ] Marmoset Toolbag 3.04 Beta, export to glTF

http://www.cgchannel.com/2018/03/marmoset-releases-toolbag-3-04-in-beta/

**Comment by pjcozzi** [comment]
* [ ] [fbx-gltf-pipeline](https://github.com/MozillaReality/fbx-gltf-pipeline) by @robertlong

**Comment by pjcozzi** [comment]
* [ ] [gltfviewer](https://github.com/mkeblx/gltfviewer) - glTF model viewer for VR/AR - by @mkeblx

**Comment by pjcozzi** [comment]
* [ ] Meta glTF import, http://www.metavision.com/solidworks

**Comment by pjcozzi** [comment]
* [ ] [gltf-content-hash](https://www.npmjs.com/package/gltf-content-hash) - by @robertlong

**Comment by pjcozzi** [comment]
* [ ] [gltf-component-data](https://www.npmjs.com/package/gltf-component-data) - by @robertlong

**Comment by pjcozzi** [comment]
* [ ] [gltf-unlit-generator](https://npm.taobao.org/package/gltf-unlit-generator) - by @robertlong

**Comment by SamuelTS** [comment]
I'm happy to announce release of my PBR extension for SketchUp :)

My extension includes a Material Editor that allows to specify metalness, roughness, normal map, etc. directly in SketchUp desktop software. Then, SketchUp model can be viewed in a Web Viewport. (Thanks notably to @CentaurMare and @pissang for their tools.)

Project is open-source and available here: https://github.com/SamuelTS/SketchUp-PBR-Plugin

**Comment by pjcozzi** [comment]
Looks great @SamuelTS, congrats!  Please open a PR to add it to https://github.com/KhronosGroup/glTF/blob/master/README.md#converters-and-exporters

**Comment by pjcozzi** [comment]
* [ ] [gltf-simple](https://www.npmjs.com/package/gltf-simple) viewer by @dakom

**Comment by pjcozzi** [comment]
* [ ] [maya-glTF](https://github.com/matiascodesal/maya-glTF) exporter by @matiascodesal

**Comment by pjcozzi** [comment]
* [ ] [GLTFKit](https://github.com/warrenm/GLTFKit ) by @warrenm

**Comment by pjcozzi** [comment]
*  [ ] kenney.nl glTF game assets via https://twitter.com/KenneyNL/status/981114126960398337

**Comment by pjcozzi** [comment]
* [ ] Sketchfab Unreal importer via https://twitter.com/kentbarber/status/976522999665909760

**Comment by pjcozzi** [comment]
* [ ] x-model via https://twitter.com/keithclarkcouk/status/976235040496594944

**Comment by pjcozzi** [comment]
* [ ] glTF converter: FBX, Collada (DAE), OBJ and MTL, glTF v1 and v2, three.js JSON format, 3MF, AMF, PCD, PLY via https://twitter.com/looeeeb/status/974299188224643078

**Comment by pjcozzi** [comment]
* [ ] 8th Wall AR glTF Viewer via http://www.euroinvestor.com/news/2018/03/21/8th-wall-is-first-to-allow-gltf-20-link-sharing-for-faster-3d-and-ar-app-development/13788179

**Comment by pjcozzi** [comment]
* [ ] [rollup-plugin-gltf](https://www.npmjs.com/package/rollup-plugin-gltf) by @bengsfort

**Comment by pjcozzi** [comment]
* [ ] [Oculus Home glTF import](https://www.oculus.com/blog/welcome-home-introducing-new-features-coming-to-the-oculus-platform-on-rift/)

**Comment by pjcozzi** [comment]
* [ ] Cheetah3D via https://twitter.com/Cheetah3D/status/985137299901034496

**Comment by dakom** [comment]
> gltf-simple viewer by @dakom

FYI I trashed that and created a new repo [pure3d](https://github.com/dakom/pure3d) - for right now it's just my "kick-the-tires-and-learn-stuff" playground so might not be worth adding to the list here, but if so it should replace `gltf-simple` above.

Of course I'm happy to let you know if/when it has proper gltf support so we can tick the box :)

**Comment by modelconverter** [comment]
Please add Model Converter 
 http://modelconverter.com to the list. Free online converter to glTF 2.0 for OBJ, COLLADA DAE, and  FBX format. Thanks.

**Comment by pjcozzi** [comment]
* [ ] Peaks Viewer, https://twitter.com/PeeksIo/status/987028250361782272

**Comment by pjcozzi** [comment]
* [ ] AR studio glTF import via  https://twitter.com/iamsbtron/status/991428229146730496

**Comment by pjcozzi** [comment]
* [ ] Magnum 2018.4 glTF importer, http://blog.magnum.graphics/announcements/2018.04/

**Comment by pjcozzi** [comment]
* [ ] Dust3D export, https://github.com/huxingyi/dust3d, by @huxingyi via https://github.com/KhronosGroup/glTF/issues/456#issuecomment-385579881

**Comment by pjcozzi** [comment]
* [ ] [gltf-loader-2](https://github.com/MONOGRID/gltf-loader-2) - webpack loader for .gltf files by @daniele-pelagatti

**Comment by pjcozzi** [comment]
* [ ] [glTFast](https://github.com/atteneder/glTFast) - glTF binary runtime loading library for Unity - by @atteneder via https://twitter.com/tteneder/status/995470799938170880

**Comment by pjcozzi** [comment]
* [ ] "model loader in my deferred PBR engine" via https://twitter.com/KamikaziUk/status/995629467765788672

**Comment by peted70** [comment]
Just added this DirectX 11 sample for viewing glTF files https://github.com/Microsoft/glTF-DXViewer . Have added pull requests to link from the glTF and the WebGL PBR sample repos.

**Comment by pjcozzi** [comment]
@najadojo - **love** it! 😁 We'll add it to the main README.md eventually, but feel free to open a PR for it any time.

**Comment by pjcozzi** [comment]
* [ ] [gltf2-blender-importer](https://github.com/julienduroure/gltf2-blender-importer) by @julienduroure

**Comment by pjcozzi** [comment]
* [ ] Meshroom VR via https://twitter.com/meshroomvr/status/1010055479035400192

**Comment by Ybalrid** [comment]
I'm working on an Ogre 2.x GLTF 2.0 loader here https://github.com/Ybalrid/Ogre_glTF

It's still WIP and not ready for being actually used, so I wouldn't put it on the list just yet

**Comment by pjcozzi** [comment]
@Ybalrid that is fantastic, please keep us posted on your progress!

* [ ] Ogre 2.x GLTF 2.0 loader (above)

**Comment by pjcozzi** [comment]
* [ ] Google ARCore Sceneform Beta glTF import, https://developers.google.com/ar/develop/java/sceneform/import-assets

**Comment by donmccurdy** [comment]
- [ ]  Maya glTF 2.0 Exporter https://github.com/kashikacojp/glTF-Maya-Exporter

**Comment by Ybalrid** [comment]
@pjcozzi hey! So, my gltf loader plugin for Ogre v2-1 is working. 

I don't really know where to put it on the master list to submit a pull request. Should it be under C++ (as it is an addon for a C++ library called OGRE - a 3D rendering engine) or under Game Engines. 

Ogre is *not* a game engine, it only does rendering. However, it is often cited as being one, because it is quite large, easilly extensible, and generally provide you with an application framework, and has a resource management system you would want in a game engine...

So, what do you think? :smile:

**Comment by donmccurdy** [comment]
I'd be OK with renaming that section. Perhaps `Native Engines` or `Game and Rendering Engines`?

Also, looks like we're missing Unreal in that list

- [ ] Unreal Engine (https://forums.unrealengine.com/development-discussion/content-creation/1415582-the-new-gltf-import-supported-in-4-19-preview-is-awesome-experimentation)

**Comment by Ybalrid** [comment]
Between your two propositions, `Game and Rendering Engines` sounds better (at least to me). We could potentially put there other rendering-only engines/libraries (like, e.g. Open Scene Graph, if somebody makes a glTF loader for them) under that category.

**Comment by Ybalrid** [comment]
@donmccurdy  @pjcozzi  see PR #1392 regarding the Ogre glTF loader :smile:

**Comment by pjcozzi** [comment]
* [ ] [Magic Leap](https://magic-leap.reality.news/news/magic-leap-mobile-companion-app-revealed-ios-android-supported-0186248/)

**Comment by pjcozzi** [comment]
* [ ] [px](https://github.com/pplux/px/blob/master/README_px_render.md) by @pplux

via https://twitter.com/pplux/status/1025437253286486017

**Comment by donmccurdy** [comment]
- [ ] Embed glTF models in AMP HTML pages: https://github.com/ampproject/amphtml/tree/master/3p/3d-gltf
- [ ] Wave Engine: https://twitter.com/WaveEngineTeam/status/1025391889615400961
- [ ] Substance Painter adds glTF _import_ (in addition to previously available export) https://www.allegorithmic.com/blog/substance-painter-summer-2018

**Comment by pjcozzi** [comment]
* [ ] Vizard 6 via https://twitter.com/glTF3D/status/1027218362265415680

**Comment by pjcozzi** [comment]
* [ ] [Google Filament Sceneform accepts glTF](https://developers.google.com/ar/develop/java/sceneform/import-assets)

**Comment by pjcozzi** [comment]
* [ ] Microsoft glTF to USD - https://twitter.com/kaceylefty/status/1028531700958023680

**Comment by pjcozzi** [comment]
* [ ] Microsoft glTF SDK - https://github.com/Microsoft/glTF-SDK

**Comment by pjcozzi** [comment]
* [ ] glTF viewer for Android - https://play.google.com/store/apps/details?id=com.vispolygon.gltfshowcase.alpha

**Comment by pjcozzi** [comment]
* [ ] Speedernet Sphere - https://www.einnews.com/pr_news/458123693/speedernet-announces-3d-gltf-integration-in-sphere-360-vr-software-that-delivers-in-webvr-at-siggraph-2018

**Comment by pjcozzi** [comment]
* [ ] [glTF for Rhino](https://github.com/mcneel/glTF-Rhino) by @mcneel

**Comment by donmccurdy** [comment]
- [ ] [InstaLOD](https://instalod.com/availability/)
- [ ] [Hypar](https://pypi.org/project/hypar/)

**Comment by bboltz** [comment]
glTF 2.0 importer/exporter plugin for Ultimate Unwrap3D
https://www.unwrap3d.com

**Comment by pjcozzi** [comment]
* [ ] Blog post (I am not modifying the main README.md until #1423 is merged): [glTF is a Huge Deal for VR + AR Creation: Here's Why We're Embracing It](https://www.worldviz.com/post/gltf-is-a-huge-deal-for-vr-ar-heres-why-were-embracing-it)

**Comment by BeRo1985** [comment]
- [ ] [PasGLTF](https://github.com/BeRo1985/pasgltf) by @BeRo1985 GLTF 2.0 loader&writer library for the Object Pascal ecosystem (Delphi and FreePascal) together with a GLTF 2.0 base-spec-core feature-complete viewer tool as code example. [Video showcase of a meanwhile older version of the PasGLTF viewer example](https://www.youtube.com/watch?v=i8AOAUP8KsM)

**Comment by Ziriax** [comment]
@pjcozzi First experimental MacOS support for [Maya2glTF](https://github.com/WonderMediaProductions/Maya2glTF) (in the `cmake` branch)

**Comment by pjcozzi** [comment]
* [ ] [gltf-viewer](https://github.com/tojacob/gltf-viewer) by @tojacob

**Comment by pjcozzi** [comment]
* [ ] Vectary - glTF to USDZ - https://www.vectary.com/3d-modeling-how-to/how-to-create-usdz-file-for-ar-online - via https://twitter.com/milangladis/status/1039920802999795719

**Comment by pjcozzi** [comment]
* [ ] [gltf2scn](https://github.com/gimlids/gltf2scn) - glTF to SceneKit by @gltf2scn

**Comment by pjcozzi** [comment]
* [ ] WorldViz glTF importer via https://twitter.com/worldviz/status/1041752376489652224

**Comment by pjcozzi** [comment]
* [ ] Creature Update glTF Export via @kestrelm, https://github.com/KhronosGroup/glTF/issues/456#issuecomment-423786224

**Comment by donmccurdy** [comment]
- [ ] KeyShot 8 (Pro edition only) adds glTF export. (http://www.cgchannel.com/2018/08/sneak-peek-keyshot-8/)

**Comment by pjcozzi** [comment]
* [ ] Houdini glTF import/export, https://twitter.com/donrmccurdy/status/1045009666479017985, via @donmccurdy

**Comment by pjcozzi** [comment]
* ARCore Sceneform glTF import via https://twitter.com/donrmccurdy/status/1045449802081742848

**Comment by pjcozzi** [comment]
* [ ] Cheetah3D via https://twitter.com/cheetah3d/status/1050426971933560832

**Comment by donmccurdy** [comment]
- [ ] Magneticraft for Minecraft: https://www.reddit.com/r/feedthebeast/comments/9b47qf/magneticraft_update_250_1122/

**Comment by pjcozzi** [comment]
* [ ] Ogre_glTF via https://twitter.com/ybalrid/status/1058428451559034881. @ybalrid

**Comment by pjcozzi** [comment]
* [ ] Autodesk Forge import and potentially export.  via https://twitter.com/ipetrbroz/status/1065635259646132226.  @petrbroz

**Comment by pjcozzi** [comment]
Not software, but formats:

* [ ] [3D Tiles](https://github.com/AnalyticalGraphicsInc/3d-tiles/)

> Specification for streaming massive heterogeneous 3D geospatial datasets

* [ ] [VRM - 3D Avatar File Format for VR](https://dwango.github.io/en/vrm/)

> “VRM” is a file format for handling 3D humanoid avatar (3D model) data for VR applications. It is based on glTF 2.0. Anyone is free to use it. In addition, a standard implementation (UniVRM) in c# that can import and export VRM file in Unity is released as open source.

**Comment by pjcozzi** [comment]
* [ ] glTF import in [Systems Tool Kit (STK)](http://help.agi.com/stk/Content/vo/glTFmodel.htm) via @emackey

**Comment by pjcozzi** [comment]
* [ ] JanusVR, https://twitter.com/bai0/status/1067928010219606016, via @jbaicoianu

**Comment by pjcozzi** [comment]
* [ ] [Ventuz](https://www.ventuz.com/support/help/latest/WhatsNew.html?itm=1) - real-time motion-graphics content development

**Comment by CentaurMare** [comment]
glTF thumbnail provider for Windows: I've attached to issue #1167
I'm not sure where it should be permanently based.

My glTF exporter for Sketchup has over 3000 users now... was a big jump when facebook announced support for the GLB format.
https://extensions.sketchup.com/en/content/gltf-exporter
https://extensions.sketchup.com/en/content/gltf-import

**Comment by pjcozzi** [comment]
* [ ] [Rapid Compact](https://darmstadt-graphics.com/rapidcompact/) import/export via @mlimper

**Comment by pjcozzi** [comment]
* [ ] [HRE :: Hybrid (Clustered Forward/Deferred) Rendering Engine](https://github.com/Angelo1211/HybridRenderingEngine) by @Angelo1211 imports glTF 2.0 using AssImp

**Comment by pjcozzi** [comment]
* [COMSOL Multiphysics version 5.4 glTF export](https://www.comsol.com/blogs/how-to-export-and-share-your-3d-result-plots-as-gltf-files/)

**Comment by pjcozzi** [comment]
* [ ] [BIMData.io](http://www.bimdata.io/en/bim-data-en/) glTF import

**Comment by pjcozzi** [comment]
* [ ] [vue-gltf-viewer](https://github.com/sajjad-shirazy/vue-gltf-viewer) - Preview for glTF 2.0 models in WebGL using three.js - by @sajjad-shirazy

**Comment by pjcozzi** [comment]
* [ ] SimLab has trial and paid glTF [importers](https://extensions.sketchup.com/en/content/simlab-gltf-importer-sketchup) and [exporters](https://extensions.sketchup.com/en/content/simlab-gltf-exporter-sketchup) for SketchUp

**Comment by pjcozzi** [comment]
* [ ] Kashika [glTF Exporter for Maya](https://twitter.com/kashikacojp/status/1053084994120253442)

**Comment by pjcozzi** [comment]
* [ ] "The #WaveEngine 3.0 low level graphics API on #Directx12 backend is loading #GLTF models now" via https://twitter.com/jcant0n/status/1091322586640932864. @jcant0n

**Comment by pjcozzi** [comment]
* [ ] cgltf via https://twitter.com/j66k/status/1096141947239043077 by @jkuhlmann

**Comment by pjcozzi** [comment]
* [ ] Ashes 3D importer via https://twitter.com/but0n97/status/1096679170816409600 by @but0n

**Comment by pjcozzi** [comment]
* [ ] [Point105-AR](https://www.point105ar.com) cloud-based glTF converter via https://github.com/KhronosGroup/glTF/issues/456#issuecomment-464978596 by @jeyashankher

**Comment by donmccurdy** [comment]
- [ ] [The Wild](https://thewild.com/) VR design collaboration tool [added support for uploading glTF models as Spaces](https://docs.thewild.com/release-notes/release-notes-the-wild-09). [Via twitter](https://twitter.com/TheWildXR/status/1097926424474906630).

**Comment by mgsx-dev** [comment]
- [ ]  [gdx-gltf](https://github.com/mgsx-dev/gdx-gltf) LibGDX glTF addon by @mgsx-dev

**Comment by pjcozzi** [comment]
* [ ] [webviz](https://github.com/cruise-automation/webviz) glTF import

**Comment by pjcozzi** [comment]
* [ ] Golang glTF reader, writer and validator by @qmuntal via https://twitter.com/matsuu_zatsu/status/1098927234742710272

**Comment by pjcozzi** [comment]
* [ ] [Noesis](http://richwhitehouse.com/index.php?content=inc_projects.php&showproject=91) glTF importer and exporter

**Comment by donmccurdy** [comment]
- [ ] glTF exporter for Cinema 4D in Maxon Labs: https://labs.maxon.net/?p=3360

**Comment by pjcozzi** [comment]
* [ ] [SharpGLTF](https://github.com/vpenades/SharpGLTF): glTF support for .NET by @vpenades via https://github.com/KhronosGroup/glTF/issues/456#issuecomment-468358498

**Comment by pjcozzi** [comment]
* [ ] Esri glTF support via https://twitter.com/GIS_Bandit/status/1102986135922401281 and @DerekLaw

**Comment by pjcozzi** [comment]
* [ ] [Tech Soft 3D HOOPS Exchange 2019 glTF importer](https://architosh.com/2019/03/hoops-exchange-2019-supports-autodesk-revit-for-aec-workflows/)

**Comment by pjcozzi** [comment]
* [ ] [Plattar's Unity3D GLTF2 Exporter toolchain](https://github.com/Plattar/gltf-exporter) by @DavidArayan.  Built on UnityGLTF.

CC @sbtron @bghgary

**Comment by pjcozzi** [comment]
* [ ] [Xiratech drag & drop glTF viewer](http://www.pasamson.net/xiratech/)

**Comment by pjcozzi** [comment]
* [ ] [VoxEdit](https://www.voxedit.io/) glTF export via https://twitter.com/VoxEdit/status/1106250644468453376

**Comment by pjcozzi** [comment]
* [ ] [javafx-gltf-loader](https://github.com/the-zucc/javafx-gltf-loader) by @the-zucc

**Comment by pjcozzi** [comment]
* [ ] [Goopy](https://goopy.io/index.php?mact=CGBlog,cntnt01,detail,0&cntnt01articleid=7&cntnt01returnid=49) via https://twitter.com/goopyAR/status/1108020864895717376

**Comment by pjcozzi** [comment]
* [ ] [AMD Prorender glTF export](https://community.amd.com/community/radeon-pro-graphics/blog/2018/11/08/radeon-prorender-transforms-rendering-workflows-with-interactive-viewport-denoising)

**Comment by Siccity** [comment]
[Unity GLTFUtility](https://github.com/Siccity/GLTFUtility)

**Comment by pjcozzi** [comment]
* [@vxna/gltf-loader](https://github.com/vxna/gltf-loader) - an opinionated webpack loader for glTF files by @vxna

**Comment by pjcozzi** [comment]
* [ ] [amp-3d-gltf](https://amp.dev/documentation/examples/components/amp-3d-gltf/?format=websites) import

**Comment by pjcozzi** [comment]
* [ ] [SharpGLTF](https://github.com/vpenades/SharpGLTF) - glTF support for .NET - by @vpenades

**Comment by pjcozzi** [comment]
* [ ] [glTF import in luma.gl](https://luma.gl/#/examples/core-examples/gltf) by @ibgreen and @georgios-uber

**Comment by georgios-uber** [comment]
> * [ ]  [glTF import in luma.gl](https://luma.gl/#/examples/core-examples/gltf) by @ibgreen and @georgios-uber

Do we need to do something?

**Comment by emackey** [comment]
@georgios-uber If you'd like to be listed in the main glTF README.md file, a PR would be most welcome.

**Comment by pjcozzi** [comment]
* [ ] [DiligentSamples](https://github.com/DiligentGraphics/DiligentSamples/tree/master/Samples/GLTFViewer) viewer by @DiligentGraphics

**Comment by pjcozzi** [comment]
* [ ] [CGI Studio](https://www.youtube.com/watch?v=uD3D-LLxgMw)

**Comment by pjcozzi** [comment]
* [ ] [Esri CityEngine](https://doc.arcgis.com/en/cityengine/latest/whats-new/cityengine-release-notes.htm) glTF import and export, and curated gallery of glTF models (replaces OBJ) via @chri7928 and team

**Comment by pjcozzi** [comment]
* [ ] [cgtrader](https://www.cgtrader.com/3d-models/ext/gltf) has 654 glTF models

**Comment by pjcozzi** [comment]
* [ ] [UniVRM](https://github.com/vrm-c/UniVRM) includes formerly separate [UniGLTF](https://github.com/ousttrue/UniGLTF), glTF 2.0 importer and exporter for Unity 5.6 or later, via @ousttrue

**Comment by pjcozzi** [comment]
* [ ] [Wisp](https://github.com/TeamWisp/WispRenderer) - a real-time ray-tracing framework for game developers - glTF import - by @VZout and team - via https://twitter.com/wisprenderer/status/1129340789031497728

**Comment by pjcozzi** [comment]
* [ ] [meshoptimizer](https://github.com/zeux/meshoptimizer) glTF import and export.  By @meshoptimizer.

**Comment by pjcozzi** [comment]
* [ ] [VTK and ParaView](https://blog.kitware.com/introducing-gltf-2-0-support-in-vtk-and-paraview/)

**Comment by pjcozzi** [comment]
* [ ] Apple's [usdzconvert](https://developer.apple.com/augmented-reality/quick-look/) converts obj/gltf/fbx/abc to usdz.  Also see the WWDC 2019 slides: https://developer.apple.com/videos/play/wwdc2019/602/

**Comment by pjcozzi** [comment]
* [ ] [amethyst](https://github.com/amethyst/amethyst) - Data-oriented and data-driven game engine written in Rust - by @ebkalderon and team

**Comment by pjcozzi** [comment]
* [ ] [NexTech](https://www.globenewswire.com/news-release/2019/06/11/1866787/0/en/NexTech-Announces-Technological-Breakthrough-Enabling-Dynamic-3D-Web-AR-Renderings-of-Complicated-Objects.html) - AR using glTF

**Comment by pjcozzi** [comment]
* [ ] [EF EVE](https://ef-eve.com/platform/?filterformat=gltf) 3D asset repo with glTF downloads

**Comment by pjcozzi** [comment]
* [ ] [Diligent Engine](http://diligentgraphics.com/) - glTF import

**Comment by pjcozzi** [comment]
* [ ] [RoomVolumeDirectShape](https://thebuildingcoder.typepad.com/blog/2019/06/room-volume-gltf-generator.html) by @jeremytammik - Revit C# .NET add-in that creates DirectShape elements representing room volume

**Comment by weegeekps** [comment]
Here are some examples of the "Are we...?" pattern sites that the Rust community uses.

* https://areweguiyet.com/
* http://arewegameyet.com/
* https://areweasyncyet.rs/
* https://www.arewewebyet.org/

I'm particularly fond of the designs for "Are we game yet?" and "Are we GUI yet?" Both provide an ecosystem matrix allowing you to drill down into specific areas in order to find what you need.

**Comment by pjcozzi** [comment]
* [ ] Various glTF to USDZ tools / discussion:
   * https://www.designar.io/usdz-converter
   * https://www.vectary.com/3d-modeling-news/usdz-converter-create-usdz-files-for-free-online-with-vectary/
   * https://darmstadt-graphics.com/doc/cli/06-exporting-to-usdz/index.html
   * https://blendersushi.blogspot.com/2018/09/convert-gltf-into-usdz_3.html
   * https://github.com/GoogleWebComponents/model-viewer/issues/278
   * https://github.com/TimvanScherpenzeel/gltf-to-usdz-research

**Comment by pjcozzi** [comment]
* [ ] Related: KTX (and KTX2?)
   * https://github.com/syoyo/tinygltf/tree/tinyktx
   * https://github.com/DeanoC/tiny_ktx

via https://twitter.com/syoyo/status/1147045127816302592 

by @DeanoC and @syoyo

**Comment by pjcozzi** [comment]
* [ ] [gltf-insight](https://github.com/lighttransport/gltf-insight) - C++11 based glTF 2.0 data insight tool - by @Ybalrid

**Comment by Ybalrid** [comment]
>     * [ ]  [gltf-insight](https://github.com/lighttransport/gltf-insight) - C++11 based glTF 2.0 data insight tool - by @Ybalrid

Ah ah, I've been spotted again by @pjcozzi on twitter I presume :wink:

Gloal of this application is to make both a mesh viewer (currently partially support the metal roughness PRB shader, I need to fix SRGB/gamma stuff and environment lighting) and a light "manipulation" tool for glTF assets.

Notably "manipulation" is focused around animation data. It's hard to just "tweak a little thing" because data is in the middle of a binary buffer.

I'm the main developer behind this (at least at the time of writing) but this porject is own by Light Transport Inc. (@lighttransport is @syoyo 's company). Unsurprisingly It's built using Tiny_glTF :smile: 

It's under heavy developement (started in May). Once it's more usable, I think this project deserve to be added to the master list too, but today is probably too soon  😅

**Comment by pjcozzi** [comment]
@Ybalrid thanks for all the details and for your contributions to the glTF ecosystem.  Please ping us when this is ready to showcase! 😄

**Comment by pjcozzi** [comment]
* [ ] [AMD's Radeon Cauldron](https://twitter.com/NThibieroz/status/1153660000306388996) - SDK with glTF import for DX12 and Vulkan prototyping via @aguaviva

**Comment by pjcozzi** [comment]
* [ ] [Autodesk BIM 360 glTF import](https://knowledge.autodesk.com/support/bim-360/learn-explore/caas/CloudHelp/cloudhelp/ENU/BIM360D-Document-Management/files/GUID-0E5E18BC-343A-47FC-B921-BDD1A40DA9F4-html.html)
* [ ] [SimLab GLTF Exporter for Autodesk® Revit®](https://apps.autodesk.com/RVT/en/Detail/Index?id=531407844766561060&appLang=en&os=Win64) on the Autodesk App Store
* [ ] [glTF Exporter by Griffel Studio](https://apps.autodesk.com/RVT/en/Detail/Index?id=8451869436709222290) on the Autodesk App Store

**Comment by pjcozzi** [comment]
* [ ] Google's [USD from glTF](https://github.com/google/usd_from_gltf) by

**Comment by pjcozzi** [comment]
* [ ] [glTFast](https://github.com/atteneder/glTFast) - glTF binary runtime loading library for Unity -  by @atteneder

**Comment by pjcozzi** [comment]
* [ ] Forge glTF model viewer by @wolfgangfengel and team via https://twitter.com/wolfgangengel/status/1167556362731024384

**Comment by pjcozzi** [comment]
* [ ] MacOS Quick Look open source project, https://twitter.com/syoyo/status/1173089611876188160?s=12
*[ ] Elevationapi.com, https://twitter.com/xavierfischer/status/1172623546448789506?s=12

**Comment by vpenades** [comment]
Hi, I've just added an [example](https://github.com/vpenades/SharpGLTF/tree/master/examples/MonoGameScene) to [SharpGLTF](https://github.com/vpenades/SharpGLTF), showcasing realtime loading and rendering with MonoGame.

I think it can be useful to people guessing how to load and render animated characters.

**Comment by zellski** [comment]
Wearing my Facebook hat, I've put together a MIT-licensed tool called [glTFVariantMeld](https://github.com/facebookincubator/glTFVariantMeld/) for generating "variational" assets, using the `FB_material_variants` extension from https://github.com/KhronosGroup/glTF/pull/1681.

It's under ongoing development, but a command-line interface is available now.

**Comment by javagl** [comment]
@weegeekps  and @pjcozzi  

I have added the latest projects mentioned here to the glTF-Project-Explorer data file via [this commit](https://github.com/KhronosGroup/glTF-Project-Explorer/commit/d09fbb9217738c8c95d3dd685c6c7ae156faecbf) (some had already been in there) 

The lists/tools mentioned in 
- *Various glTF to USDZ tools / discussion* (https://github.com/KhronosGroup/glTF/issues/1058#issuecomment-509429821 ) and
-  *Related: KTX (and KTX2?)* ( https://github.com/KhronosGroup/glTF/issues/1058#issuecomment-511228064 )

seem to either focus *only* on USD/KTX, or be extensions to tools that have already been in the list, so I guess there should not be any action for these.

**Comment by pjcozzi** [comment]
* [ ] Magnum Engine 2019.10 , https://twitter.com/gltf3d/status/1191419360797233154?s=12

**Comment by pjcozzi** [comment]
* [ ] [acl-gltf](https://github.com/nfrechette/acl-gltf) - Animation Compression Library glTF toolkit.  glTF extension to follow.  By @nfrechette.

**Comment by pjcozzi** [comment]
* [ ] [Gru](https://github.com/ashikns/Gru) - glTF runtime importer for Unity3D by @ashikns

**Comment by pjcozzi** [comment]
* [ ] [Apple Reality Converter](https://developer.apple.com/news/?id=01132020a&1578956733)

> drag-and-drop common 3D file formats, such as .obj, .gltf and .usd, to view the converted USDZ result,

**Comment by MESLewis** [comment]
- [ ] [SimpleGLTF2](https://github.com/MESLewis/SimpleGLTF2)

Port of glTF-Sample-Viewer to Java

I've been working on this for a few weeks now, I wanted to learn more about the glTF format and I'm planning on writing some about the process of writing a glTF importer to help others. 

Needs a lot of clean up but should be useful to some people already. Its still being actively developed, the end goal is to have a clean api to use glTF in an existing project.

**Comment by pjcozzi** [comment]
Thanks for sharing @MESLewis!  Please let us know about your articles as well.

**Comment by pjcozzi** [comment]
* [ ] [Amazon Sumerian glTF export](https://aws.amazon.com/about-aws/whats-new/2020/04/amazon-sumerian-supports-exporting-scenes-to-gltf/)

**Comment by pjcozzi** [comment]
* [ ] [Arvizio XR platform adds glTF support](https://www.auganix.org/arvizio-announces-enhancements-to-its-enterprise-xr-platform-and-increased-data-format-support/)
